diff --git a/Cargo.toml b/Cargo.toml
index b0a508a7..10527050 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -13,14 +13,14 @@ use_camera = ["freya/use_camera"]
 hot-reload = ["freya/hot-reload"]
 
 [patch.crates-io]
-# dioxus = { git = "https://github.com/DioxusLabs/dioxus", rev = "7beacdf9c76ae5412d3c2bcd55f7c5d87f486a0f" }
-# dioxus-rsx = { git = "https://github.com/DioxusLabs/dioxus", rev = "7beacdf9c76ae5412d3c2bcd55f7c5d87f486a0f" }
-# dioxus-core-macro = { git = "https://github.com/DioxusLabs/dioxus", rev = "7beacdf9c76ae5412d3c2bcd55f7c5d87f486a0f" }
-# dioxus-hooks = { git = "https://github.com/DioxusLabs/dioxus", rev = "7beacdf9c76ae5412d3c2bcd55f7c5d87f486a0f" }
-# dioxus-signals = { git = "https://github.com/DioxusLabs/dioxus", rev = "7beacdf9c76ae5412d3c2bcd55f7c5d87f486a0f" }
-# dioxus-core = { git = "https://github.com/DioxusLabs/dioxus", rev = "7beacdf9c76ae5412d3c2bcd55f7c5d87f486a0f" }
-# dioxus-hot-reload = { git = "https://github.com/DioxusLabs/dioxus", rev = "7beacdf9c76ae5412d3c2bcd55f7c5d87f486a0f" }
-# dioxus-router = { git = "https://github.com/DioxusLabs/dioxus", rev = "7beacdf9c76ae5412d3c2bcd55f7c5d87f486a0f" }
+dioxus = { git = "https://github.com/DioxusLabs/dioxus", branch = "main" }
+dioxus-rsx = { git = "https://github.com/DioxusLabs/dioxus", branch = "main" }
+dioxus-core-macro = { git = "https://github.com/DioxusLabs/dioxus", branch = "main" }
+dioxus-hooks = { git = "https://github.com/DioxusLabs/dioxus", branch = "main" }
+dioxus-signals = { git = "https://github.com/DioxusLabs/dioxus", branch = "main" }
+dioxus-core = { git = "https://github.com/DioxusLabs/dioxus", branch = "main" }
+dioxus-hot-reload = { git = "https://github.com/DioxusLabs/dioxus", branch = "main" }
+dioxus-router = { git = "https://github.com/DioxusLabs/dioxus", branch = "main" }
 
 [workspace.dependencies]
 freya = { path = "crates/freya", version = "0.2" }
